pkg/cli/cvd.go

```diff
@@ -143,17 +143,30 @@ func (c *cvdCreator) Create() ([]*hoapi.CVD, error) {
 
 func (c *cvdCreator) createCVDFromLocalBuild() ([]*hoapi.CVD, error) {
 	vars, err := GetAndroidEnvVarValues()
-	if err != nil {
-		return nil, fmt.Errorf("error retrieving Android Build environment variables: %w", err)
-	}
-	names, err := ListLocalImageRequiredFiles(vars)
-	if err != nil {
-		return nil, fmt.Errorf("error building list of required image files: %w", err)
-	}
-	if err := verifyCVDHostPackageTar(vars.HostOut); err != nil {
-		return nil, fmt.Errorf("invalid cvd host package: %w", err)
+	var names []string
+	if err == nil {
```

Comment:
The code pattern for this function should be roughly:
```
cvd, err := createCVDFromBuiltSource(...)
if errors.Is(err, NotInAndroidSourceErr) {
  return createCVDFromCurrentDir(...)
}
return cvd, err
```
I think encapsulating both flows into their own functions will increase readability.